### ISSUE TITLE
as a funder, i want hunters to have to stake funds to 'start work' on an issue, so i know they're serious about it

### ISSUE THREAD

**Issue by owocki** [gitcoinco/web#1893]
### User Story
as a funder, i want users to have to stake funds to 'start work' on an issue, so i know they're serious about it

[comment]: # (As a <user type>, I want to <task> so that <goal>.)

### Why Is this Needed
- to reduce abandonment rate
- permissionless issues are preventing stale issues from being completed, because users who "take" the issue but don't actually do anything completely blocks out the issue for anyone else.

[comment]: # (Describe the problem and why this task is needed. Provide description of the current state, what you would like to happen, and what actually happen)
*Summary*:

### Description

there would be a third option for permission types
* permissionless
* approval required
* staking required

in a v2, we could change the staking amount too.

when a bounty is paid out, the funder will have the option of returning the stake and/or donating the stake to a charity. (or burning it maybe?)

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*: Feature

### Current Behavior
[comment]: # (Describe what actually happened.)
no staking 

### Expected Behavior
[comment]: # (Describe what you expected to happen.)
staking 

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)
see descriptoin

**Comment by owocki** [comment]
@PixelantDesign @vs77bb any feedback on this idea?

**Comment by PixelantDesign** [comment]
I like the idea. Seems like staking is a layer that could apply to  permissionless and approval required.

I see it as a separate characteristic of a bounty. Not as a permission.

If I set a project to permissionless, because I don't want to have to approve a contributor, but still want to know they're serious about the task. A stake would make sense here.

If I set a project to approval required, I want to ensure the candidate has a good track record, as added security that the contributor would deliver, staking would make sense here as well.
